docs/guides/first_pipeline.md

```diff
@@ -1,26 +1,9 @@
 # Getting started
 
-!!! note
-
-    To execute the pipeline locally, you must have docker compose, Python >=3.8 and Git
-    installed on your system.
-
-!!! note
-
-    For Apple M1/M2 ship users: <br>
-    - Make sure that Docker uses linux/amd64 platform and not arm64. <br>
-    - In Docker Dashboards’ Settings<Features in development, make sure to uncheck `Use containerid for pulling and storing images`.
+Check out the [installation guide](installation.md) to get started with Fondant.
 
 For demonstration purposes, we provide sample pipelines in the Fondant GitHub repository. A great starting point is the pipeline that loads and filters creative commons images. To follow along with the upcoming instructions, you can clone the [repository](https://github.com/ml6team/fondant) and navigate to the `examples/pipelines/filter-cc-25m` folder.
 
-This pipeline loads an image dataset and reduces the dataset to png files. For more details on how you can build this pipeline from scratch, check out our [guide](guides/build_a_simple_pipeline.md).
-
-Install Fondant by running:
-
-```
-pip install fondant
-```
-
 Clone the Fondant GitHub repository
 
 ```
```

Comment:
Link at the bottom of the page "[guide](guides/build_a_simple_pipeline.md)." down. Must send to "Building your own pipeline.

Reply:
Good catch, thanks